src/backend/joanie/tests/swagger/admin-swagger.json

```diff
@@ -5911,7 +5972,7 @@
                     }
                 }
             },
-            "StateEnum": {
+            "State693Enum": {
```

Comment:
We should add 
```"OrderStateEnum": "joanie.core.enums.ORDER_STATE_CHOICES",```
to https://github.com/openfun/joanie/blob/main/src/backend/joanie/settings.py#L298 to override this generated name.

Reply:
Oh thanks for the tips !